piptools/_compat/click_compat.py

```diff
@@ -0,0 +1,5 @@
+import click
+from pip._vendor.packaging.version import Version
+
+CLICK_VERSION = Version(click.__version__).major
```

Comment:
I don't think the var name corresponds to its contents. Either add that it's major or only store the version.